modules/lx-autoscale/watchmaker-lx-autoscale.template.cfn.yaml

```diff
@@ -881,6 +894,16 @@ Resources:
 
               #!/bin/bash -xe
 
+              # Extend any available LVM PVs
+              if [[ -x $( which pvs ) ]]
+              then
+                LVMPVS=($(pvs --noheadings -o pv_name))
```

Comment:
Mind providing an example of this output as a pr comment, so can get a better idea of what this is doing?

Reply:
[Updated](https://github.com/plus3it/terraform-aws-watchmaker/pull/92/commits/4b7d3d11da4a67b15d8ed754fb78f3d5fd1c98ac)